src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -40,6 +40,7 @@ public class NuspecReader : NuspecCoreReaderBase
         private const string ExcludeFlags = "exclude";
         private const string LicenseUrl = "licenseUrl";
         private const string Repository = "repository";
+        private const string Icon = "icon";
```

Comment:
You are adding a const string but not using it.

Reply:
Removed.